pymeasure/instruments/microtest/__init__.py

```diff
@@ -0,0 +1 @@
+from .microtest8761 import Microtest8761
```

Comment:
Please also add Microtest to the [instruments init file](https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html#updating-the-init-file). :-)